app/src/main/java/chat/rocket/android/onboarding/ui/OnboardingActivity.kt

```diff
@@ -0,0 +1,112 @@
+package chat.rocket.android.onboarding.ui
+
+import android.content.res.Resources
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.support.v4.view.ViewPager
+import chat.rocket.android.R
+import chat.rocket.android.authentication.ui.AuthenticationActivity
+import chat.rocket.android.onboarding.adapter.OnboardingPagerAdapter
+import chat.rocket.android.onboarding.presentation.PageTransformerPresenter
+import chat.rocket.android.onboarding.utils.Utils
+import chat.rocket.android.util.extensions.setVisible
+import kotlinx.android.synthetic.main.activity_onboarding.*
+
+class OnboardingActivity : AppCompatActivity(){
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+
+        val adapter = OnboardingPagerAdapter(supportFragmentManager)
+        view_pager.adapter = adapter
+        pageIndicator.initViewPager(view_pager)
+
+        button_next.setOnClickListener{
+            if (view_pager.currentItem != adapter.count-1){
```

Comment:
Missing space in `adapter.count-1` shoud be `adapter.count - 1`. The same happens on a lot of places on this PR.

Reply:
@divyanshub024 You can try `ctrl + alt + shift + L` for formating selected text or whole file in android studio :smile: